app/views/layouts/application.html.slim

```diff
@@ -12,7 +12,17 @@ html dir="ltr" lang="en-GB" class="govuk-template"
     = tag.meta name: "viewport", content: "width=device-width, initial-scale=1"
     = tag.meta property: "og:image", content: asset_path("images/govuk-opengraph-image.png")
     = tag.meta name: "theme-color", content: "#0b0c0c"
-    = tag.meta name: "robots", content: "noindex"
+    ruby:
+      index_prod = FeatureFlags.enabled?(:index_production, without_session_data: true) ? "" : "noindex"
+    = tag.meta name: "robots", content: index_prod.to_s
+
+    ruby:
+      canoncical_link = if ENV["PRIMARY_HOST"].present?
+                          "https://check-your-client-qualifies-for-legal-aid.service.gov.uk#{url_for(only_path: true, protocol: 'https')}"
```

Comment:
Rather than hardcode the URL here, would it make sense to use `"https://#{ENV["PRIMARY_HOST"]}"`?

Reply:
Also the `protocol: https` bit is redundant, I think, as the `url_for` isn't building the bit with the protocol in it.